unified-runtime/source/adapters/level_zero/v2/command_list_manager.cpp

```diff
@@ -158,19 +158,14 @@ ur_result_t ur_command_list_manager::appendRegionCopyUnlocked(
 }
 
 wait_list_view ur_command_list_manager::getWaitListView(
-    const ur_event_handle_t *phWaitEvents, uint32_t numWaitEvents,
-    ur_event_handle_t additionalWaitEvent) {
+    const ur_event_handle_t *phWaitEvents, uint32_t numWaitEvents) {
```

Comment:
@staniewzki is using this API in https://github.com/intel/llvm/pull/18179/files#diff-57b5ceadf73f501ea8d07f9837583b61b45ca22129f3aacdeda1fb0b582a9990R735-R736.